content/faq/how-to-publish.md

```diff
@@ -0,0 +1,39 @@
+## How to publish content on LBRY
+
+Advanced users: All of the steps below are for the LBRY app.  For additional details of how to conduct any of these actions through the cli please see https://lbryio.github.io/lbry/cli/
+
+### How to create a channel
+1. Download the LBRY app from lbry.io/get. If you already have the application and it prompts you to update, run the update to be on the most recent version.
+2. Load the application and  click the `Publish` button.
+3. In the `Channel Name` section click the dropdown and select `New Channel` and  declare the name you would like for your channel.  For more details on different channel types, see https://lbry.io/faq/naming
+4. Once your name is selected, there is a `Deposit` section that is below.  It requires a minimum bid of .1 LBC.   See more on deposits here: naming.md.  If you would ever like to abandon your claim there is a guide below in `Abandoning a claim`.
```

Comment:
We don't really have a minimum or a way to calculate one right now (it's like a TX fee, it varies) - but if you had to give one, I'd say 0.0001 LBC to be safe. 

"See more on deposits here: naming.md". - naming faq url?

Reply:
I decided not to include anything on linking to abandoning claims since the naming faq is already referenced, it will be in my commit that I'm pushing soon.